docs/API/LorisRESTAPI_v0.0.3.md

```diff
@@ -728,3 +728,63 @@ The JSON object is of the form:
     "Value" : string
 }
 ```
+
+# 5.0 DICOM Data
+
+Like the imaging data, the DICOM data mostly lives in the `/candidates/$CandID/$Visit` 
+portion of the REST API namespace, but is defined in a separate section of this 
+document for clarity purposes.
+
+## 5.1 Candidate DICOMs
+```
+GET /candidates/$CandID/$Visit/dicoms
+```
+
+A GET request to `/candidates/$CandID/$Visit/dicoms` will return a JSON object of
+all the raw DICOM data which have been acquired for that visit. It will return an 
+object of the form:
+
+```js
+{
+    "Meta" : {
+        "CandID" : $CandID,
+        "Visit" : $VisitLabel,
+    },
+    "DicomTars" : 
+        [{
+        "Tarname" : "DCM_yyyy-mm-dd_ImagingUpload-hh-mm-abc123.tar",
+        "SeriesInfo" :
+            [{
+            "SeriesDescription" : "MPRAGE_ipat2",
+            "SeriesNumber" : "2",
+            "EchoTime" : "2.98",
+            "SeriesUID" : "1.2.3.4.1107",
+            },
+            {
+            "SeriesDescription" : "BOLD Resting State",
+            "SeriesNumber" : "5",
+            "EchoTime" : "30",
+            "SeriesUID" : "3.4.5.6.1507",
+            }],
+        "Tarname" : "DCM_yyyy-mm-dd_ImagingUpload-hh-mm-def456.tar",
+        "SeriesInfo" :
+            [{
+            "SeriesDescription" : "MPRAGE_ipat2",
+            "SeriesNumber" : "2",
+            "EchoTime" : "2.98",
+            "SeriesUID" : "1.7.8.9.1296",
+            }],
+    }],    
+}
+```
+
+## 5.2 Tar Level Data
+```
+GET /candidates/$CandID/$VisitLabel/dicoms/$Tarname
+```
+
+Returns a `tar` file which contains a `.meta` and a `.log` text files, 
+and a `.tar.gz` of the raw DICOM data as acquired during the candidate
+scanning session, and as retrieved from `/candidates/$CandID/$Visit/dicoms`.
```

Comment:
might be useful to specify that the raw data should be `>`'d into a file on the user's / client's system? depends on how this plans to be used

Reply:
I changed the word `returns` to `downloads` as this is what this will do.